pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -280,11 +300,23 @@ func (lb *ListenerBuilder) buildInboundChainConfigs() []inboundChainConfig {
 				Protocol:   i.ServicePort.Protocol,
 			}
 
+			var bindAddr, extraBindAddrs string
```

Comment:
should we be using the NewWildcardAndLocalHost here?

Reply:
Sure, u r right!